pyanaconda/modules/common/base/base.py

```diff
@@ -23,9 +23,9 @@
 from abc import ABC
 from locale import setlocale, LC_ALL
 
+from dasbus.loop import EventLoop
 from pykickstart.errors import KickstartError, KickstartParseWarning
 
-from pyanaconda.core.event_loop import EventLoop
```

Comment:
I don't think we should take EventLoop from a library. We are the main application the event loop should go opposite way. We should provide event loop to the library.

In case that we would have two libraries with their event loop we would end up with broken scenario because event loops can't be started twice.

Reply:
TODO: Use `create_main_loop` instead.

Reply:
Fixed.